src/pages/documentation/migrate-to-android.md

```diff
@@ -79,11 +79,12 @@ Kotlin
 
 Some of the APIs available in previous major versions of the Mobile SDK for Android are now deprecated. You can choose to replace the deprecated APIs in your code with the alternative APIs in the latest version, as described in the table below:
 
+<InlineAlert variant="info" slots="text"/>
+
+The `registerExtension` API for each extension is deprated in the latest mobile SDK. You don't need to register extensions seperately, now you can call `MobileCore.registerExtensions` instead. See the [Update SDK initialization](#update-sdk-initialization) section for more details.
```

Comment:
typo: deprated -> deprecated, seperately-> separately

Reply:
Oh, and one more - "... in the latest version of the Mobile SDK."